.changelog

```diff
@@ -4,46 +4,6 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
-# Last packaged: 19-04-2021
+# Last packaged: 2021-05-17
 
-- Documented Prometheus TLS support in the Administration Guide and the
-  Salt Guide.
-- Fixed certificate names in Red Hat Client Registration chapter of the
-  Client Configuration Guide
-- Use the term "Product Migration" instead of "SP Migration".
-- Documented retracted patches in Administration Guide.
-- Documented filter templates for Content Lifecycle Management in
-  Administration Guide 
-- Documented Ansible Feature in Administration and Reference Guide 
-- Completely rewritten Autoinstallation sections in the Client Configuration
-  Guide.
-- Update OpenSCAP section in the Administration Guide for usability.
-- Documented sharing patches between cloned channels in CLM of the
-Administration Guide.
-- A zip archive now replaces tarball when downloading all PDF documents from the index page
-- Documented apply custom state at highstate in Salt Guide
-- Adds troubleshooting information for bootstrapping
-  new CentOS 6 clients (bsc#1181932)
-- Adds clarification for which GPG keys are required when
-  bootstrapping SLES15 and Debian clients (bsc#1181695)
-- Finds and updates all Public Cloud content. These are the most important changes or news:
-  * Disables Public Cloud Requirements, Public Cloud Setup
-  * New: page in  the Client Configuration Guide, Registering clients on a public cloud
-  * New book: Quick Start: Public Cloud overview
-- Clarify Action Chains instructions in Administration
-  Guide (bsc1182180)
-- Add external network configuration to Retail Guide
-- Clarify backup instructions in Administration Guide (bsc#1182945)
-- Document ARM64 support in Retail
-- Updates PAM authentication procedure in Administration Guide
-- Adds additional dependencies for Debian client registration
-  in Client Configuration Guide (bsc#1183649)
-- Adds instructions to delete vendor channels in
-  Administration Guide (bsc#1183293)
-- Adds troubleshooting entry to Administration Guide for
-  registering traditional RHEL 7 clients (bsc#1182700)
-- Adds missing Virtualization Network tab to System Details
-  section in Reference Guide
-- Adds SLE Micro and openSUSE MicroOS clients in the Client Configuration Guide
-- Adds troubleshooting multiple Red Hat CDN certificates to Client
-  Configuration Guide (bsc#1184448)
+- Fixed URL of API documentation
```

Comment:
Sorry, not OK, you are missing the bsc :-)

Reply:
bsc for the API URL?  there is non, afaics.  Or is it about the other changelog entries?  We have the (bad) habit to reset the .changelog file when everything is transferred to the .changes of the package(s).  But I do not mind keeping the complete .changelog here.

Reply:
Ah, crap. I only created a github card :-D

Ignore my comment.

Reply:
ok ;)